src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedContainer.cs

```diff
@@ -221,5 +221,7 @@ internal override IEnumerable<Cci.SecurityAttribute> GetSecurityInformation()
         internal sealed override NamedTypeSymbol AsNativeInteger() => throw ExceptionUtilities.Unreachable;
 
         internal sealed override NamedTypeSymbol NativeIntegerUnderlyingType => null;
+
+        internal override bool HasPossibleWellKnownCloneMethod() => false;
```

Comment:
>HasPossibleWellKnownCloneMethod [](start = 31, length = 31)

I think it would be better to keep overrides of this API and ```IsRecord``` next to each other. Also, we prefer implementing abstract APIs in leaf types, that is why ```IsRecord``` not overridden here.